.husky/pre-commit

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
```

Comment:
Ten precommit sprawi, ze zadziala prettier, ale czy zmiany zaaplikowane przez prettiera zostana dolaczone do tego samego commita? Czy trzeba bedzie je "docommitowac"?

Reply:
@kacgrzes sprawdziłem, trzeba "dokomitować". Może po `yarn prettier:write` dorzucić` git add . ` ` git commit` z message np. "prettier format" ?

Reply:
no wlasnie to powinno odbywac sie inaczej